### PR TITLE
🇩🇪 Improve German translation for the 21 Tweaks (How not to diet)

### PR DESCRIPTION
**Pull request by caugner**
:us: ~~This is a work in progress.~~ If you are fluent in German, feel free to review my changes so far.
:de: ~~Dieser PR ist noch nicht fertig.~~ Bitte reviewe aber gerne schon einmal meine bisherigen Änderungen.

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -594,82 +594,82 @@
 
     <string name="twenty_one_tweaks">21 Optimierungen</string>
```
Comment (caugner) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="twenty_one_tweaks">21 Tweaks</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -594,82 +594,82 @@
...
-    <string name="daily_tweaks_history">Daily Tweaks Geschichte</string>
-    <string name="weight_history">Gewichtsverlauf</string>
-    <string name="no_weights_recorded">Sie haben noch keine Gewichte erfasst!</string>
-    <string name="task_loading_weights_history_title">Gewichtsverlauf wird geladen</string>
-    <string name="morning">Morgen</string>
-    <string name="evening">Abend</string>
-    <string name="weight">Gewicht</string>
-    <string name="tweak_group_meal">Bei jeder Mahlzeit</string>
+    <string name="daily_tweaks_history">Täglicher Optimierungsverlauf</string>
+    <string name="weight_history">Körpergewichtsverlauf</string>
```
Comment (caugner) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="weight_history">Gewichtsverlauf</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -594,82 +594,82 @@
...
+    <string name="daily_hydrate">Genug trinken</string>
+    <string name="daily_deflourdiet">Mehl durch Vollkorn ersetzen</string>
+    <string name="daily_frontload">Kalorien früh verzehren</string>
+    <string name="daily_timerestrict">Essen zeitlich beschränken</string>
+    <string name="exercise_timing">Zeiten sportlicher Aktivitäten optimieren</string>
+    <string name="weigh_twice">Zweimal täglich wiegen</string>
+    <string name="complete_intentions">Verhaltensvorsätze befolgen</string>
+    <string name="nightly_fast">Fasten nach 19.00 Uhr</string>
+    <string name="nightly_sleep">Genug schlafen</string>
+    <string name="nightly_trendelenburg">Mit Mild Trendelenburg experimentieren</string>
```
Comment (caugner) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="nightly_trendelenburg">Mit leichter Trendelenburg-Lage experimentieren</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -594,82 +594,82 @@
...
-    <string name="morning">Morgen</string>
-    <string name="evening">Abend</string>
+    <string name="morning">Morgens</string>
+    <string name="evening">Abends</string>
     <string name="weight">Gewicht</string>
-    <string name="tweak_group_meal">Bei jeder Mahlzeit</string>
+    <string name="tweak_group_meal">Zu jeder Mahlzeit</string>
     <string name="tweak_group_daily">Jeden Tag</string>
-    <string name="tweak_group_dailydoses">Nehmen Sie Ihre täglichen Dosen</string>
+    <string name="tweak_group_dailydoses">Nehmen Sie eine tägliche Portition</string>
```
Comment (caugner) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="tweak_group_dailydoses">Nehmen Sie eine tägliche Portion</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -594,82 +594,82 @@
...
+    <string name="daily_frontload_short">Kalorien früh verzehren</string>
+    <string name="daily_timerestrict_short">Essen zeitlich beschränken</string>
+    <string name="exercise_timing_short">Bewegung zeitlich abstimmen</string>
+    <string name="weigh_twice_short">Zweimal täglich wiegen</string>
+    <string name="complete_intentions_short">Verhaltensvorsätze befolgen</string>
+    <string name="nightly_fast_short">Fasten nach 19 Uhr</string>
+    <string name="nightly_sleep_short">Genug schlafen</string>
+    <string name="nightly_trendelenburg_short">Mit leichter Trendelenburg-Lage experimentieren</string>
+    <string name="meal_water_text">Fördern Sie Ihren Stoffwechsel, indem Sie vor jeder Mahlzeit zwei Tassen kühles oder kaltes Wasser (ohne Geschmack) zu sich nehmen.</string>
+    <string name="meal_negcal_text">Beginnen Sie jede Mahlzeit als ersten Gang mit einem Apfel, eine Suppe mit grüner Ampel oder einem Salat mit weniger als 100 Kalorien pro Tasse.</string>
```
Comment (caugner) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="meal_negcal_text">Beginnen Sie jede Mahlzeit als ersten Gang mit einem Apfel, einer Suppe mit grüner Ampel oder einem Salat mit weniger als 100 Kalorien pro Tasse.</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -594,82 +594,82 @@
...
-    <string name="meal_negcal">Vorladen mit "Negative Calorie" -Nahrungsmitteln</string>
-    <string name="meal_vinegar">Essig einarbeiten (2 TL zu jeder Mahlzeit)</string>
-    <string name="meal_undistracted">Genießen Sie ungestörte Mahlzeiten</string>
-    <string name="meal_twentyminutes">Befolgen Sie die 20-Minuten-Regel</string>
+    <string name="about_tweak">Über die Optimierung</string>
+    <string name="meal_water">Mahlzeit mit kaltem Wasser beginnen</string>
+    <string name="meal_negcal">Mahlzeit mit kalorienarmer Nahrung beginnen</string>
+    <string name="meal_vinegar">Essig hinzufügen (2 TL zu jeder Mahlzeit)</string>
+    <string name="meal_undistracted">Ungestört essen</string>
+    <string name="meal_twentyminutes">20-Minuten-Regel gefolgen</string>
```
Comment (caugner) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="meal_twentyminutes">20-Minuten-Regel befolgen</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 7]
```diff
@@ -594,82 +594,82 @@
...
+    <string name="nightly_fast_short">Fasten nach 19 Uhr</string>
+    <string name="nightly_sleep_short">Genug schlafen</string>
+    <string name="nightly_trendelenburg_short">Mit leichter Trendelenburg-Lage experimentieren</string>
+    <string name="meal_water_text">Fördern Sie Ihren Stoffwechsel, indem Sie vor jeder Mahlzeit zwei Tassen kühles oder kaltes Wasser (ohne Geschmack) zu sich nehmen.</string>
+    <string name="meal_negcal_text">Beginnen Sie jede Mahlzeit als ersten Gang mit einem Apfel, einer Suppe mit grüner Ampel oder einem Salat mit weniger als 100 Kalorien pro Tasse.</string>
+    <string name="meal_vinegar_text">Trinken Sie Essig niemals pur. Würzen Sie stattdessen Ihre Mahlzeiten oder gönnen Sie sich einen Beilagensalat mit einem beliebigen süßen und herzhaften Essig. Wenn Sie den Essig trinken möchten, mischen Sie ihn mit einem Glas Wasser und spülen Sie anschließend Ihren Mund mit Wasser, um Ihren Zahnschmelz zu schützen.</string>
+    <string name="meal_undistracted_text">Sehen Sie nicht fern oder spielen mit Ihrem Handy, während Sie essen. Setzen Sie einen Häkchen für jede Mahlzeit, die Sie ohne Ablenkung zu sich nehmen.</string>
```
Comment (caugner) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="meal_undistracted_text">Sehen Sie nicht fern oder spielen mit Ihrem Handy, während Sie essen. Setzen Sie ein Häkchen für jede Mahlzeit, die Sie ohne Ablenkung zu sich nehmen.</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 8]
```diff
@@ -594,82 +594,82 @@
...
+    <string name="meal_undistracted_text">Sehen Sie nicht fern oder spielen mit Ihrem Handy, während Sie essen. Setzen Sie ein Häkchen für jede Mahlzeit, die Sie ohne Ablenkung zu sich nehmen.</string>
+    <string name="meal_twentyminutes_text">Dutzende Studien haben gezeigt, dass die Kalorienaufnahme verringert werden kann, wenn wir die Zeit des Essens im Mund verlängern. Und zwar unabhängig davon, ob wir dies durch Essen mit zäherer Konsistenz, längeres Kauen, kleinere Portionen im Mund oder langsameres Essen erreichen. Verlängern Sie also die Dauer der Mahlzeit auf mindestens zwanzig Minuten, damit Ihre natürlichen Sättigungssignale ihre volle Wirkung entfalten können. Und wie? Indem Sie Lebensmittel auswählen, deren Verzehr länger dauert, und sie so essen, dass sie möglichst lange im Mund verbleiben. Denken Sie an sperrigere, härtere, kaubarere Lebensmittel in kleineren, gut gekauten Bissen.</string>
```
Comment (caugner) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="meal_twentyminutes_text">Dutzende Studien haben gezeigt, dass die Kalorienaufnahme verringert werden kann, wenn wir die Zeitspanne des Essens im Mund verlängern. Und zwar unabhängig davon, ob wir dies durch Essen mit zäherer Konsistenz, längeres Kauen, kleinere Portionen im Mund oder langsameres Essen erreichen. Verlängern Sie also die Dauer der Mahlzeit auf mindestens zwanzig Minuten, damit Ihre natürlichen Sättigungssignale ihre volle Wirkung entfalten können. Und wie? Indem Sie Lebensmittel auswählen, deren Verzehr länger dauert, und sie so essen, dass sie möglichst lange im Mund verbleiben. Denken Sie an sperrigere, härtere, kaubarere Lebensmittel in kleineren, gut durchgekauten Portionen.</string>
```

**Comment by slavick** [comment]
Thanks for the PR!

**Comment by marc-medley** [comment]
@jigbits The translation process has now evolved such that an online application is available for anyone interested to be part of the _Daily Dozen App Translation Volunteer Team_. The team works with Steven Litrov (@salitrov) the Global Volunteer Director. We are currently in need of some German translation talent.

You, or others that you know, are welcome to apply:

* [https://nutritionfacts.org/volunteer/app-translation-volunteer/](https://nutritionfacts.org/volunteer/app-translation-volunteer/)

The current translations for both the Android and Apple Daily Dozen app can be viewed in the `daily-dozen-localization` repository in a `.tsv` spreadsheet form:

* [nutritionfactsorg/daily-dozen-localization repository](https://github.com/nutritionfactsorg/daily-dozen-localization)

Thank you for your patience with the Daily Dozen App Translation process evolution.

--Marc

**Pull request merged**